variables.tf

```diff
@@ -9,96 +9,16 @@ variable "subnets" {
   default     = []
 }
 
-variable "create_security_group" {
-  type        = bool
-  default     = true
-  description = "Set `true` to create and configure a new security group. If false, `associated_security_group_ids` must be provided."
-}
-
-variable "associated_security_group_ids" {
+variable "ingress_cidr_blocks" {
+  description = "List of IPv4 CIDR ranges to use on all ingress rules"
   type        = list(string)
   default     = []
-  description = <<-EOT
-    A list of IDs of Security Groups to associate the created resource with, in addition to the created security group.
-    These security groups will not be modified and, if `create_security_group` is `false`, must provide all the required access.
-    EOT
 }
 
-variable "allowed_security_group_ids" {
+variable "additional_security_groups" {
+  description = "A list of security group IDs to assign to the Redis."
```

Comment:
Description change required. additional_security_groups are added to ingress rule, not assigned to the cluster.